app2/src/lib/transfer/shared/components/ChainAsset/ChainSelector.svelte

```diff
@@ -171,6 +171,9 @@ const filterChainsByTokenAvailability = (chains: Array<Chain>): Array<ChainWithA
             onNone: () => chains.map(chain => Tuple.make(chain, false)),
             onSome: ({ baseToken, sourceChain }) =>
               chains.map(destinationChain => {
+                if (destinationChain.testnet === true) {
+                  return Tuple.make(destinationChain, true)
+                }
```

Comment:
Can you add a small comment to explain that for testnets we don't aply bucket-based ratelimiting?